src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const mainPrice = 40;
+  let totalPrice = days * mainPrice;
+  const smallestDiscount = 20;
+  const biggestDiscount = 50;
+
+  if (days >= 7) {
+    totalPrice -= biggestDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalPrice -= smallestDiscount;
+  }
```

Comment:
it is difficult to understand where we have changed totalPrice and what will returned